wrapper/lib/wrapper.js

```diff
@@ -3,6 +3,14 @@ import Client from '@zenginehq/post-rpc-client'
 
 var plugin = {}
 
+Function.prototype.curry = Function.prototype.curry || function () {
+  const fn = this
+  const args = Array.prototype.slice.call(arguments)
+  return function () {
+    return fn.apply(this, args.concat(Array.prototype.slice.call(arguments)))
+  }
+}
```

Comment:
Ew gross. Extending the Function prototype?

Reply:
lmao, maybe i'll change it after ironing out the other kinks in the znData